index.html

```diff
@@ -30,5 +30,199 @@ <h1 class="header header-1" id="job">I'm a software developer</h1>
                 </ul>
             </address>
         </header>
+        <main class="container" id="main-container">
+            <div class="header-container" id="header-2-container">
+                <h2 class="header header-2" id="recent">My Recent Works</h2>
```

Comment:
- [ ] OPTIONAL 
You used lots of `id`'s for naming your elements. It is a best practice to use `class`s over `id`s as this will keep your code DRY.

Reply:
**OPTIONAL**
As said by the previous TSE, you have many `id's` to tag your element, which is not a good practice. In the future, you have to use `class` for a cleaner code.

**REQUIRED**
The "select work" section should be wrapped inside a `section` tag. Please try and fix that.